### DISCUSSION TITLE
1.4.0

### DISCUSSION THREAD

**Discussion by leeyuentuen** [pypolestar/polestar_api#67] [Release Discussion]
## What's Changed
* Release 1.4.0 by @leeyuentuen in https://github.com/leeyuentuen/polestar_api/pull/66
* raise error exception on polestarApi(https://github.com/leeyuentuen/polestar_api/pull/66/commits/6e4c184ee6ece7157f1a300eba7436ff92700940)
* Remove unit in labels (https://github.com/leeyuentuen/polestar_api/pull/66/commits/a460906836e0a25e7c5f4243922d29c4822df235) https://github.com/leeyuentuen/polestar_api/issues/49
* add not supported sensors(https://github.com/leeyuentuen/polestar_api/pull/66/commits/eaea41392ce292dae7704a6dcc1279720ff83bb9)
* add comments, try catch http timeout issue(https://github.com/leeyuentuen/polestar_api/pull/66/commits/9dd5b555e39565d20bed850bbe6f3327a2fa2033)
* remove sensor key with meter, watt, amps,...(https://github.com/leeyuentuen/polestar_api/pull/66/commits/175b334d046579cad9c6476944d2bb98e87cdda1)
* Danish translation (https://github.com/leeyuentuen/polestar_api/pull/66/commits/4d997c183e41fded9e7cbfd766348e46d485adc8)

**Full Changelog**: https://github.com/leeyuentuen/polestar_api/compare/1.3.0...1.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/leeyuentuen/polestar_api/releases/tag/1.4.0'>1.4.0</a>.</em>

**Comment by ohaue** [comment]
This release seems to be very stable and it works very well for me.
Thank you @leeyuentuen for your work :-)
